src/reporters/test-reporter.js

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+import type {QuestionOptions} from './types.js';
+import BaseReporter from './base-reporter.js';
+
+export default class TestReporter extends BaseReporter {
```

Comment:
Nice! I'd base this on `BufferReporter` which really helps with testing. I'll do this and merge the patch afterward.